File: Selenium/Project/CRM_Activity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:
   # Navigate to the URL
    driver.get("https://alchemy.hguy.co/crm/")

    #Verify url and print to console
    headerText = driver.find_element(By.XPATH,"*//div/img[@alt='SuiteCRM']").get_attribute("src")
    print("Header url is : ",headerText)
    assert headerText=="https://alchemy.hguy.co/crm/themes/default/images/company_logo.png?v=cK7kLsY0ftg72ZVHTYUT_g"

    #closing browser
    driver.quit()

File: Selenium/Project/CRM_Activity2.java
package project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.Reporter;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity2 {
	WebDriver driver;
	
	@BeforeClass
	  public void beforeClass() {
		  driver = new ChromeDriver();
		  
	  }

	  
  @SuppressWarnings("deprecation")
@Test
  public void getHearderImageUrl() {
	  
	//Launch application
	  Reporter.log("Launching application |");
	  driver.get("https://alchemy.hguy.co/crm/");
	  
	  //Verify url and print to console
	  String headerText = driver.findElement(By.xpath("*//div/img[@alt='SuiteCRM']")).getAttribute("src");
	  System.out.println("The header url is : "+headerText);
	  Assert.assertEquals(headerText,"https://alchemy.hguy.co/crm/themes/default/images/company_logo.png?v=cK7kLsY0ftg72ZVHTYUT_g");
	  
	  
  }
  
  @AfterClass
  public void afterClass() {
	  driver.quit();
  }
}
